### ISSUE TITLE
SAML - Get the Service Provider Certificate

### ISSUE THREAD

**Issue by misilot** [hackmdio/codimd#1886]
Hello,

Is there a way to expose the SAML SP Certificate via the metadata? That way I can register it with our Shibboleth server?

Thanks!

**Comment by Yukaii** [comment]
Yes, our CodiMD exposes the SAML SP certificate through the metadata endpoint. You can access it at `⁠https://[our-codimd-domain]/auth/saml/metadata` and use this URL to register our service with your Shibboleth IdP server. The metadata XML contains all the necessary information, including our SP certificate.

https://github.com/hackmdio/codimd/blob/b4ec353fcd5cfdee333373087ea9ff386f2fbea7/lib/auth/saml/index.js#L93-L96
